tools/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd ../cypher-codemirror
+
+yarn run build-demo
+
+cd demo
+
+git init
+
+git remote add origin git@github.com:neo4j-contrib/cypher-editor.git
+
+git checkout -b gh-pages
+
+git add --all
+
+git commit -m "Deploy website"
+
+git push origin gh-pages
+
+echo -e "\e[32mSuccess!"
```

Comment:
In my example I had all commands combined with `&&`.

This is done for a reason. If any of the command will fail at any point, then script will not proceed to next command. Similarly how boolean operators work in programming languages.

In your case if for some reason you can't `cd` into the folder, then error will be printed and script will continue. Leading to adding remote to some random location and push code somewhere.